be/src/exec/schema_scanner/schema_be_metrics_scanner.cpp

```diff
@@ -0,0 +1,136 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "exec/schema_scanner/schema_be_metrics_scanner.h"
+
+#include "agent/master_info.h"
+#include "exec/schema_scanner/schema_helper.h"
+#include "gutil/strings/substitute.h"
+#include "runtime/string_value.h"
+#include "types/logical_type.h"
+#include "util/metrics.h"
+#include "util/starrocks_metrics.h"
+
+namespace starrocks {
+
+SchemaScanner::ColumnDesc SchemaBeMetricsScanner::_s_columns[] = {
+        {"BE_ID", TYPE_BIGINT, sizeof(int64_t), false},
+        {"NAME", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"LABELS", TYPE_VARCHAR, sizeof(StringValue), false},
+        {"VALUE", TYPE_BIGINT, sizeof(int64_t), false},
+};
+
+SchemaBeMetricsScanner::SchemaBeMetricsScanner()
+        : SchemaScanner(_s_columns, sizeof(_s_columns) / sizeof(SchemaScanner::ColumnDesc)) {}
+
+SchemaBeMetricsScanner::~SchemaBeMetricsScanner() = default;
```

Comment:
1. How long does it collect the metrics from BE?
2. If the metrics need to be persisted, after the FE restarted?

Reply:
>  How long does it collect the metrics from BE?
pretty fast in my test, like in the example.
`1 row in set (0.03 sec)`

> If the metrics need to be persisted, after the FE restarted?
Metrics don't persist currently.